lib/angular_components.dart

```diff
@@ -11,7 +11,7 @@ import 'src/all_components.dart';
 export 'src/all_components.dart';
 
 /// A convenience list of all Directives exposed by this package.
-const List<Type> materialDirectives = const [
+const List<dynamic> materialDirectives = const [
```

Comment:
What in here isn't a `Type`?

Reply:
DisplayNameRenderDirective changed in this update. It is now a "functional directive".
https://github.com/dart-lang/angular_components/pull/153/files#diff-a8788133997a1100324e3684b3b3e6c3

Reply:
Ah! Cool...